### DISCUSSION TITLE
nfqws-keenetic + pi.hole

### DISCUSSION THREAD

**Discussion by KeDP72** [Anonym-tsk/nfqws-keenetic#117] [Q&A]
Доброго, уважаемые!
Подскажите кто сталкивался, может, или подробно разбирается.
Есть в моей сети сервер pi.hole который успешно боролся с рекламой, после установки nfqws pi.hole игнорируется, хотя по прежнему установлен как днс сервер для домашней сети. Что смотреть, куда копать, какие варианты есть?
*Провайдерские днс игнорируются, настроен DoT. Аппарат Giga (KN-1010). nfqws использую в режиме list.
Спасибо

**Comment by zetcamp** [comment]
> Провайдерские днс игнорируются, настроен DoT

А галка с "транзитных запросов" там же, где DoH настраивается - снята?


> установлен как днс сервер для домашней сети

В настройках DHCP-сервера, верно?

П.С.
В качестве альтернативы, если такое возможно, могу предложить развернуть на кинетике или где-то рядом AdGuardHome в качестве DNS-сервера и настроить резалку рекламы на нём.

**Reply by KeDP72** [reply]
> А галка с "транзитных запросов" там же, где DoH настраивается - снята?

Галка снята очень давно.

> В настройках DHCP-сервера, верно?

Верно.

> AdGuardHome

То же думал о нем, но хотелось бы и дальше pi.hole использовать, его то же можно переустановить +unbound к нему поставить. Но хочется тут разобратся.

**Reply by zetcamp** [reply]
Тогда странно, конечно.
Если DHCP раздаёт верный DNS, то какой выхлоп на любом клиенте (у которого IP по DHCP получен или DNS вбит руками) домашней сети в "nslookup ya.ru"?
Клиент точно смотрит на pi.hole?

**Reply by KeDP72** [reply]
> Тогда странно, конечно.

Вот и я говорю - странно. Потому и написал, ибо что то не понимаю я.
Выхлоп:
> nslookup ya.ru
> ╤хЁтхЁ:  pi.hole
> Address:  192.168.1.50
> 
> Не заслуживающий доверия ответ:
> ╚ь :     ya.ru
> Addresses:  2a02:6b8::2:242
>           77.88.55.242
>           5.255.255.242
>           77.88.44.242

ДНС на клиентах везде верный, всмысле, тот что прописан в DHCP

**Reply by zetcamp** [reply]
Хорошо. А как тогда обнаружено, что  "pi.hole игнорируется" клиентами сети? По логам pi.hole? Ваером?

**Reply by KeDP72** [reply]
> как тогда обнаружено

Реклама полезла - это первое, ну и в логах pi.hole нет событий. Что то где то работает не так, будто бы "мимо".

**Reply by zetcamp** [reply]
> Реклама полезла

Тогда шарком, например, смотреть, куда обращается клиент со своими DNS-запросами.

**Reply by KeDP72** [reply]
> Тогда шарком

Пожалуй оставлю этот пердолинг на выходные, спасибо за советы, еще приду
